src/main/scala/tech/minna/utilities/PathAsString.scala

```diff
@@ -8,13 +8,8 @@ import scala.reflect.macros.blackbox
 
 object PathAsString {
 
-  @compileTimeOnly("option.~> is only intended to be used in combination with PathAsString macro")
-  implicit class PathTraverseOption[T](option: Option[T]) {
-    def ~> : T = ???
-  }
-
-  @compileTimeOnly("traversable.~> is only intended to be used in combination with PathAsString macro")
-  implicit class PathTraverseTraversableOnce[T](traversable: TraversableOnce[T]) {
+  // @compileTimeOnly("iterable.~> is only intended to be used in combination with PathAsString macro")
```

Comment:
Have not figured out how to fix this yet.
If I uncomment this, it will cause compilation error that says "iterable.~> is only intended to be used in combination with PathAsString macro" on line 45.